### PR TITLE
Disable dropdown option

### PR DESCRIPTION
**Pull request by garethdn**
This is usually not an issue with a top-level dropdown that is opened by clicking a button. In that case, we will usually disable the button, preventing the click from being registered and preventing the dropdown from opening.

For nested dropdowns, the dropdown toggle is usually bound to a `li` element, upon which the `disabled` attribute does nothing. The result is that we have no way of preventing nested dropdowns from opening.

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -82,7 +82,8 @@
         "serve": {
           "builder": "@angular-devkit/build-angular:dev-server",
           "options": {
-            "browserTarget": "demo-app:build"
+            "browserTarget": "demo-app:build",
+            "port": 4201
```
Comment (sjeansonNewswhip) [inline angular.json:86]: Out of curiosity, why is this needed ?

**Review by sjeansonNewswhip — APPROVED** [review]

File: angular.json (hunk 1 above)
Comment (garethdn) [reply angular.json:86]: I updated this because if we're running Spike locally at the same, we'll get an error when we run the Style Guide on local because port 4200 is already in use by Spike
